install/install.sh

```diff
@@ -162,6 +162,12 @@ sed -i '/noipv4ll/d' /etc/dhcpcd.conf
 # add noipv4ll
 sed -i '$ a noipv4ll' /etc/dhcpcd.conf
 
+tee -a /etc/dhcpcd.conf << END
+interface uap0
+    static ip_address=192.168.3.2/24
+    nohook wpa_supplicant
+END
```

Comment:
Based on your [last comment](https://github.com/bluerobotics/BlueOS-docker/pull/1019#issuecomment-1190183614), we can let right this for now.

Reply:
> Based on your [last comment](https://github.com/bluerobotics/BlueOS-docker/pull/1019#issuecomment-1190183614), we can let right this for now.

Did you check this: https://wiki.archlinux.org/title/dhcpcd#Hooks ?
It may be better for us to have a hook over editing the main conf file.
@rafaellehmkuhl what you think ?

Reply:
Maybe, should we bind `/etc/dhcpcd.conf` on the docker and to the necessary changes at runtime ?